Make install (cargo install) failing

`make install` on a fresh clone of `master` is failing for me with the following error

```
rustup run --install 1.39.0 cargo --color always install --force --path .
error: found a virtual manifest at `/home/farlion/code/tenx/comit-rs/Cargo.toml` instead of a package manifest
Makefile:40: recipe for target 'install' failed
make: *** [install] Error 101
```

```
$ rustup --version
rustup 1.20.2 (13979c968 2019-10-16)
```

```
$ rustc --version
rustc 1.39.0 (4560ea788 2019-11-04)
```

```
$ cargo --version
cargo 1.39.0 (1c6ec66d5 2019-09-30)
```

```
$ uname -a
Linux topbox 4.15.0-72-generic #81-Ubuntu SMP Tue Nov 26 12:20:02 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
```

--- a/Makefile
+++ b/Makefile
@@ -37,7 +37,7 @@ yarn_install:
 ## User install
 
 install:
-	$(CARGO) install --force --path .
+	$(CARGO) install --force --path cnd
 
 clean:
 	$(CARGO) clean

